### DISCUSSION TITLE
Improvement

### DISCUSSION THREAD

**Discussion by homoscribens** [homoscribens/Paper-Introduction-JAIST2022#9] [Ideas]
## このプロジェクトの改善点を思いついた方はここにコメントして下さい

良いと思ったコメントは絵文字等で反応してくれると幸いです

**Comment by homoscribens** [comment]
読み終えた論文はcloseではなくopenにしたほうがいいかも

**Comment by kan25d5** [comment]
論文のPDFをCodeに公開するのは法的もしくはコンプライアンス的にまずいかも

**Comment by homoscribens** [comment]
**Discussionへ移行**

**Comment by homoscribens** [comment]
# scrapboxも便利だが…？
## メリット
- ~~texの数式が使える~~ (Githubも対応)
- 総合的なページの書きやすさはこっち
- ページ間の関係性が分かりやすい（リンク）
- 新規ページを作りやすい（テンポがよく増やせる）

## デメリット
- **Markdownと記法が違う**
- 上記の理由で見出し機能がない
- コメント機能がない
- タグ機能はGithubのほうがわかりやすい？
- 作成中のページを判別しにくい（issuesならopen/closeがある）
